src/computedFn.ts

```diff
@@ -35,15 +35,15 @@ import { IComputedValue, computed, onBecomeUnobserved, _isComputingDerivation, i
  * @param fn 
  * @param keepAlive 
  */
-export function computedFn<T extends Function>(fn: T, keepAlive = false) {
+export function computedFn<T extends (...args: any[]) => any>(fn: T, keepAlive = false) {
```

Comment:
What about using `unknown` instead of `any`? In TS 3.5.1 they made it like a default fallback. Not in this specific case, but I think it makes sense...

https://devblogs.microsoft.com/typescript/announcing-typescript-3-5/#breaking-changes

Reply:
https://github.com/microsoft/TypeScript/blob/master/lib/lib.es5.d.ts#L1474
There are definitions of `Parameters` and `ReturnType`. They use a generic argument, that extends from `(...args: any) => any`.

https://stackoverflow.com/a/51439876/1259301
Based on this answer, in our case no difference between use `any` or `unknown`. I just use it to infer types from passed callback.

Reply:
Ok, thanks for explanation.